src/components/Dashboard/Dashboard.jsx

```diff
@@ -1,11 +1,36 @@
 import React from 'react';
 
 import Header from 'components/Header/Header';
+import IngredientCard from 'components/IngredientCard/IngredientCard';
 import SearchBox from 'components/SearchBox/SearchBox';
+import Themes from 'assets/json/themes.json';
+import Chip from 'components/Chip/Chip';
 
+import Milk from 'images/ingredients/milk.png';
+import Yogurt from 'images/ingredients/yogurt.png';
+
+// TODO: Move all the place components in their own area
 const Dashboard = () => (
   <div className="dashboard">
     <Header />
+    <IngredientCard
```

Comment:
What is your plane on deciding these themes in the future?? This is why i was suggesting that you should be getting the theme from services. How are you going to decide the them when you get the Name and Image from services?

Reply:
The theme will come from the backend. I don't know if you remember but we talked about this earlier.